shared/team-building/service-tab-bar.tsx

```diff
@@ -33,76 +33,82 @@ const HoverIcon = Styles.styled<typeof Kb.Icon, ExtraProps>(Kb.Icon)(props => ({
   color: props.isActive ? serviceIdToAccentColor(props.service) : inactiveServiceAccentColor,
 }))
 
-const ServiceIconDesktop = (props: IconProps) => (
-  <Kb.ClickableBox onClick={props.onClick} style={styles.clickableServiceIcon}>
-    <Kb.Box2 direction="horizontal" centerChildren={true} style={styles.serviceIconContainer}>
-      <Kb.WithTooltip text={serviceIdToLabel(props.service)}>
-        <HoverIcon
-          isActive={props.isActive}
-          service={props.service}
+const ServiceIconDesktop = (props: IconProps) => {
+  const showCount = props.showCount && props.service !== 'email'
```

Comment:
Wish the diff was better, I basically just added this line and used `showCount` instead of `props.shouwCount`